src/test/ui/process/signal-exit-status.rs

```diff
@@ -2,6 +2,7 @@
 // ignore-emscripten no processes
 // ignore-sgx no processes
 // ignore-windows
+// ignore-fuchsia code returned as ZX_TASK_RETCODE_EXCEPTION_KILL, FIXME (#109976)
```

Comment:
There's no Fuchsia issue here; the issue is on the standard library exposing these concepts correctly, so it belongs on this Github repo. Probably you can point directly to #58590. (Short of that, which is a major change, it's possible we could map this `std::os::unix::process::ExitStatusExt` API partly onto Fuchsia contexts.)

Btw, to link to the issue you created you want http://fxbug.dev/109976. A number starting with # refers to an issue in this Github repo.